### PR TITLE
Various GH action workflow updates

### PR DESCRIPTION
**Pull request by camargo**
* **Tickets addressed:** None
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

- Rename codeql.yml to security-scan.yml (also includes nasa scrub)
- Remove "Aerie PR" workflow since from what I can tell it's obsolete
- Capitalize workflow names
- Remove some unnecessary quotes

### CODE REVIEW

File: .github/workflows/pr.yml
Comment (Mythicaeda) [inline .github/workflows/pr.yml:1]: This file doesn't seem to be obsolete. It looks to trigger [this action](https://github.com/nasa-ammos/aerie-pr-action), which is what auto-assigns the PR opener as assignee and triggers the bot to give one approval on hotfix/docs PRs

Comment (camargo) [reply .github/workflows/pr.yml:1]: 1. Do we really need the PR opener to be assigned by this bot?
2. We only need one approver per PR now

Do we really need that action?

Comment (Mythicaeda) [reply .github/workflows/pr.yml:1]: 1. On our "Current Sprint by Teammate" view, we group by assignee. If a PR is created without a linked issue (ie this one), "assignee" is why we're able to see it under the correct person.
2. Do we never intend to go back to two approvals?

Comment (camargo) [reply .github/workflows/pr.yml:1]: 1. Yeah honestly a bot feels like overkill for assigning PRs. Most of the time we should use issues, and we can easily just assign a PR in GH projects if needed. I will say a bot would be nice for some GH Project stuff (e.g. auto moving an issue to review if you link it to a PR), but I'm waiting for GH to roll out more Project features before we go down that path.
2. Nope, not anytime soon

**Review by Mythicaeda — APPROVED** [review]

**Pull request merged**